addons/godot-xr-tools/functions/Function_Teleport.gd

```diff
@@ -40,7 +40,7 @@ export (int, LAYERS_3D_PHYSICS) var valid_teleport_mask = ~0
 # export (int, LAYERS_3D_PHYSICS) var collision_mask = 1
 
 # We don't know the name of the camera node...
-export (NodePath) var camera = null
+export var camera: NodePath = NodePath("")
 
 onready var ws = ARVRServer.world_scale
 var origin_node = null
```

Comment:
I don't know if the converter does this automatically but I remember when I did the manual conversion it really didn't like when variables meant to be used as nodes weren't typed. Maybe we should pre-empt that here and as a principle start typing these types of variables?

So in this case: `var origin_node : XROrigin`

Also the `= null` gave issues for me, it is enough to just type the node and it will default to empty correctly.

Reply:
I removed the null assignment; but I don't think node-path typing is possible in 3.x and instead requires the new 4.x format "@export_node_path(Button, TouchScreenButton) var some_button"